dnnv/nn/operations/tensor.py

```diff
@@ -221,6 +221,22 @@ def from_onnx(cls, onnx_node, *inputs):
         return cls(*inputs, axes=axes, name=onnx_node.name)
 
 
+class Split(Operation):
+    def __init__(self, x, axis, split, *, name: Optional[str] = None):
+        super().__init__(name=name)
+        self.x = x
+        self.axis = axis
+        self.split = split
+
+    @classmethod
+    def from_onnx(cls, onnx_node, *inputs):
+        attributes = {a.name: as_numpy(a) for a in onnx_node.attribute}
+        axis = attributes.get("axis")
```

Comment:
The axis attribute should have a default value.

```suggestion
        axis = attributes.get("axis", 0)
```

Reply:
Thanks! Updated with converter functionality, tests, and stylistic changes